kitchen.yml

```
@@ -1,40 +1,129 @@
+        - sh bootstrap-salt.sh -XdPbfrq -x python3 git develop
+  - name: centos-7-develop-py3
+    driver:
+      image: netmanagers/salt-develop-py3:centos-7
+      provision_command:
+        - curl -o bootstrap-salt.sh -L https://bootstrap.saltstack.com
+        - sh bootstrap-salt.sh -XdPbfrq -x python3 git develop
+  - name: fedora-30-develop-py3
+    driver:
+      image: netmanagers/salt-develop-py3:fedora-30
+      provision_command:
+        - curl -o bootstrap-salt.sh -L https://bootstrap.saltstack.com
+        - sh bootstrap-salt.sh -XdPbfrq -x python3 git develop
+  - name: opensuse-leap-15-develop-py3
+    driver:
+      image: netmanagers/salt-develop-py3:opensuse-leap-15
+      provision_command:
+        - curl -o bootstrap-salt.sh -L https://bootstrap.saltstack.com
+        - sh bootstrap-salt.sh -XdPbfrq -x python3 git develop
+      run_command: /usr/lib/systemd/systemd
+
+  ## SALT 2019.2
+  - name: debian-9-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:debian-9
+  - name: ubuntu-1804-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:ubuntu-18.04
+  - name: centos-7-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:centos-7
+  - name: fedora-30-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:fedora-30
+  - name: opensuse-leap-15-2019-2-py3
+    driver:
+      image: netmanagers/salt-2019.2-py3:opensuse-leap-15
+      run_command: /usr/lib/systemd/systemd
+
+  ## SALT 2018.3
+  - name: debian-9-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:debian-9
+  - name: ubuntu-1604-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:ubuntu-16.04
+  - name: centos-7-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:centos-7
+  - name: fedora-29-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:fedora-29
+  # TODO: Use this when fixed instead of `opensuse-leap-42`
+  # Ref: https://github.com/netmanagers/salt-image-builder/issues/2
+  # - name: opensuse-leap-15-2018-3-py2
+  #   driver:
+  #     image: netmanagers/salt-2018.3-py2:opensuse-leap-15
+  #     run_command: /usr/lib/systemd/systemd
+  - name: opensuse-leap-42-2018-3-py2
+    driver:
+      image: netmanagers/salt-2018.3-py2:opensuse-leap-42
+      run_command: /usr/lib/systemd/systemd
+
+  ## SALT 2017.7
+  - name: debian-8-2017-7-py2
+    driver:
+      image: netmanagers/salt-2017.7-py2:debian-8
+  - name: ubuntu-1604-2017-7-py2
+    driver:
+      image: netmanagers/salt-2017.7-py2:ubuntu-16.04
+  # TODO: Modify the formula to work for non-`systemd` platforms
+  - name: centos-6-2017-7-py2
+    driver:
+      image: netmanagers/salt-2017.7-py2:centos-6
+      run_command: /sbin/init
+  - name: fedora-29-2017-7-py2
+    driver:
+      image: netmanagers/salt-2017.7-py2:fedora-29
+  - name: opensuse-leap-15-2017-7-py2
+    driver:
+      image: netmanagers/salt-2017.7-py2:opensuse-leap-15
+      run_command: /usr/lib/systemd/systemd
 
 suites:
```

Can we move the `provisioner` and `verifier` sections to just above the `suites:` block?  That makes it easier to compare with the `template-formula` over time, when new changes need to be propagated across all of the `semantic-release` formulas.  I.e.:

```suggestion
provisioner:
  name: salt_solo
  log_level: info
  salt_install: none
  require_chef: false
  formula: vault
  salt_copy_filter:
    - .kitchen
    - .git

verifier:
  name: inspec
  
suites:
```

---

When moving that manually in my `diff`, it allowed me to see that there are a few more values available in `template-formula` that would be useful to bring across here, just for cross-formula consistency.  Those can be added as well but aren't necessary for merging this PR.

---

Collecting some notes from the discussion on Slack earlier.  This one about using `gid: vault` when replacing the recently deprecated `gid_from_name`.

> -|-|-
> :-:|---|---
> 20:59 | myii | @​dafydd We may need to consider https://github.com/saltstack-formulas/users-formula/issues/198#issuecomment-494927619. Or will it always be `gid: vault`?
> 21:00 | myii | I haven't looked into this in any detail, so it may not work in this situation. If it is useful, then a simple Jinja `if` block based on the `saltversioninfo` grain will probably be sufficient.
> 22:52 | dafydd | my thinking here was that a jinja `if` would not particularly promote readability - when the change i propose works in all current and dev versions
> 22:53 | myii | Sure, so it will always be `gid: vault`, never anything else?
> 22:56 | dafydd | the aim is that the user and the group have the same name, and the user's default group is the group of the same name. so as things stand, yes it will never be anything else
> 22:56 | dafydd | one could argue that the user/group name should be configurable but currently the formula follows the upstream (vault) install guidelines
> 22:57 | myii | Looking at https://github.com/saltstack-formulas/vault-formula/blob/6eddca53bff0d874406bde23014e0f829ae59995/vault/package/install.sls, there are three places that `vault` is used for user/group config. How about putting this in `defaults.yaml` and using it from the map instead?
> 22:58 | myii | This then allows for the future possibility of overriding via. the config/pillar. That doesn't need to be mentioned in `pillar.example` at the moment, though.
> 22:59 | myii | 4, if you consider `- home: /var/lib/vault` as well.
> 22:59 | dafydd | i'd like to keep the PR focused on the kitchen/travis changes - by all means we can add an issue for this stuff for further work
> 23:00 | myii | That's fine, the literal use of `vault` matches the rest of the `.sls`. And avoids the Jinja `if`.